ci/checksizes.sh

```diff
@@ -45,11 +45,11 @@ case $FULLVER in
     REVERSI_PREOPT_GZ_EXPECTEDSIZE=71000
     REVERSI_OPT_GZ_EXPECTEDSIZE=32000
     ;;
-  2.12.1)
-    REVERSI_PREOPT_EXPECTEDSIZE=492000
-    REVERSI_OPT_EXPECTEDSIZE=114000
-    REVERSI_PREOPT_GZ_EXPECTEDSIZE=66000
-    REVERSI_OPT_GZ_EXPECTEDSIZE=29000
+  2.12.2)
+    REVERSI_PREOPT_EXPECTEDSIZE=629000
+    REVERSI_OPT_EXPECTEDSIZE=144000
+    REVERSI_PREOPT_GZ_EXPECTEDSIZE=76000
+    REVERSI_OPT_GZ_EXPECTEDSIZE=33000
```

Comment:
This is single-handedly caused by https://github.com/scala/scala/commit/ce80124ca9a303a235eda16bb3533593f3701855